.github/workflows/c-cpp.yml

```diff
@@ -120,7 +134,19 @@ jobs:
           env:
             CC:  ${{ matrix.config.cc }}
             CXX: ${{ matrix.config.cxx }}
+          if: ${{ !matrix.config.name  == 'Ubuntu - Emscripten' }}
```

Comment:
I think this is not correct, it should be instead:
 `if: ${{ matrix.config.name  != 'Ubuntu - Emscripten' }}`
infact The build step run only for Emscripten. You can check in the github actions logs.

Reply:
done with https://github.com/facebook/igl/pull/72/commits/e64c1a79fa440e1e25a5a8deaae98d8d1e52e272